### DISCUSSION TITLE
Define a relationship between PK parameters of radiolabelled and non-radiolabelled molecules of the same active subtances

### DISCUSSION THREAD

**Discussion by nguyen-tranbach** [Open-Systems-Pharmacology/Forum#1853] [Models] [answered]
Dear community,

I had to fit radiolabelled and non-radiolabelled data of the same drug molecules, following alternative doses of radiolabelled and non-radiolabelled drug.

To do that I thought of creating 2 different molecules, but how do I tell PK-sim that they share the same PK parameters?

**Comment by Yuri05** [comment, marked as answer]
You need to link those parameters to the same _identification parameter_ as explained here: https://docs.open-systems-pharmacology.org/shared-tools-and-example-workflows/parameter-identification#defining-identification-parameters
